gofakes3_test.go

```diff
@@ -286,7 +286,9 @@ func TestCreateObjectMetadataHeaders(t *testing.T) {
 			t.Fatalf("Failed to parse returned expires: %s\n", err)
 		}
 
-		t.Fatalf("Expected: %s, got: %s\n", expected, got)
```

Comment:
Author of this line(@denizsurmeli)
![artworks-RtR5x6pM3ZRIS2x6-OpcCyw-t500x500](https://github.com/igungor/gofakes3/assets/52484739/dc96b108-e437-4f65-b08c-7dca2e60b5e7)